File: U4/src/EJ4.java
import java.util.Scanner;

class EJ4 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("introduce un numero, vamos a hacer su tabla del uno al diez");
        int a = sc.nextInt();
        tabla(a);

    }
    static void tabla (int a){
        for (int b = 0; b<=10;b++){
            int c = a*b;
            System.out.println(a+" x "+b+ " = " +c );

        }


    }
}

File: EjerciciosPracticar JUNIO1/src/Ev3/Kotlin/Ej4.kt
package Ev3.Kotlin

import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    println("Introduce un numero")
    val num = scanner.nextInt()
    var contador = 0;

    for (i in 1..num ){
        if (num % i == 0){
            contador++
        }
    }
    if (contador == 2){
        println("Es primo")
    }else{
        println("No es primo")
    }
}